### DISCUSSION TITLE
Java auto instrumentation is taking lot of CPU I/O time to connect during pod boot up

### DISCUSSION THREAD

**Discussion by yashrastogi21** [open-telemetry/opentelemetry-java-instrumentation#9215] [Q&A]
<img width="849" alt="image" src="https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/88483494/f7aeec2f-e895-4151-be6b-a35cacab8d89">

This is happening during the pod start and hence , facing scaling issues with this .

**Comment by trask** [comment]
hi @yashrastogi21! can you provide more context, e.g. is this I/O time or CPU time? 12 minutes to perform ~900 I/O operations, reading ~5KB seems very slow I/O performance
